ui/src/views/network/CreateSharedNetworkForm.vue

```diff
@@ -592,11 +592,12 @@ export default {
           this.formPhysicalNetworkLoading = false
         }
       }).finally(() => {
-        if (isLastNetwork) {
+        if (this.formPhysicalNetworks.length > 0 && this.isObjectEmpty(this.formSelectedPhysicalNetwork)) {
           this.form.setFieldsValue({
             physicalnetworkid: 0
           })
-          this.handlePhysicalNetworkChange(this.formPhysicalNetworks[0])
+          const network = (this.formPhysicalNetworks.length > 0) ? this.formPhysicalNetworks[0] : physicalNetwork
```

Comment:
@ravening do we still need the change in line 599 ?

Reply:
@weizhouapache no . try the change in commit caeaa1aadeea98bae1a14b733026f87c6f69dd63

if that works for you, i will merge the commits to 1

Reply:
@ravening it works without change on line 599-600.

a minor issue is, if I do not input the startip/endip/gw/netmask, I got two errors, 1 from UI, 1 from api.
![image](https://user-images.githubusercontent.com/57355700/110105173-5d67e600-7da8-11eb-8838-47a350d8ee12.png)